style.css

```diff
@@ -88,3 +97,305 @@
 .social-icons-set img {
   margin: 0 9px;
 }
+
+.works-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 80px;
+  padding: 120px 0;
+}
+
+.grid-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 24px;
+  max-width: 100%;
+  gap: 24px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 5%;
+  padding: 0 10px;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.card-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.frame-2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+}
```

Comment:
[OPTIONAL]
- [ ] Noticed that your project's images and texts are not responsible. Please use responsive values in your CSS rules, like percentages(i.e. `width: 80%`), instead of pixels.

### Your website 
![Screenshot from 2023-03-25 03-02-07](https://user-images.githubusercontent.com/91142432/227690238-8838a03b-8a9d-4073-89ab-6b3f733d0492.png)

### Figma design
![227659989-8bacf931-3a4c-49f1-bd16-0ff7a1b0209c](https://user-images.githubusercontent.com/91142432/227690256-ed20db72-2812-4961-bea0-02632eaaf3cb.png)